docs/changelog/93197.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 93197
+summary: Add null check for G1 related options and origin field
```

Comment:
Could you mention here what this change is for

Reply:
Hi! I've updated it.